document/RuleMetadataProfile.mdk

```diff
@@ -1 +1,165 @@
-# Rule Metadata Profile { #rule-metadata-profile }
\ No newline at end of file
+# Rule Metadata Profile { #rule-metadata-profile }
+
+## General
+
+It is sometimes useful for an issue management system to provide information to the end user
+about the rules that were violated by the issues in a log file.
+For example, the user might want to know whether the issue is a potential security risk,
+or where to find additional documentation for the issue.
+We refer to this information as rule metadata.
+
+It is possible for an issue management system to have built-in knowledge of
+the metadata for the rules defined by a set of well-known static analysis tools.
+It is also possible for the system to provide a means for end users to upload
+metadata for rules defined by specialized tools used in their organizations.
+But some tools enable end users to define their own rules by means of a plug-in
+mechanism.
+In this case, it is useful if the metadata for these rules can be included
+in the log file itself.
+
+The Rule Metadata Profile defines additional optional elements of the SARIF format
+that are intended for use in this scenario.
+
+## `runLog` object { #rule-metadata-profile-runLog }
+
+### General { #rule-metadata-profile-runLog-general }
+
+The Rule Metadata Profile defines one additional property in the `runLog` object (&sect;[#runLog]).
+
+### `ruleInfo` property { #ruleInfo }
+
+A `runLog` object (&sect;[#runLog]) may contain a property named `ruleInfo`
+whose value is an array of `ruleDescriptor` (&sect;[#ruleDescriptor]) objects.
+
+## `ruleDescriptor` object { #ruleDescriptor }
+
+### General { #ruleDescriptor-General }
+
+A `ruleDescriptor` object describes a discrete, potentially configurable,
+unit of analysis as implemented by an individual tool.
```

Comment:
I would just say "A `ruleDescriptor` object contains information that describes a rule." We already have a definition for "rule" in the Terms and Definitions section. I don't mind if you want to add "potentially configurable" to that definition, although I'm not sure there's a reason to do that in the context of this spec, since nothing in the spec makes use of the configurability...

... yet. When we get to the Policy Profile, we might include information about the configuration of each rule that was run. Maybe that's the time to add that to the definition.